AirSkyBoat/AirSkyBoat
# LSB Cherry-Pick: [core] Speedup of LoadNPCList() and LoadMOBList()

**_I affirm:_**


## Please enter a player-facing description

Omit. No player facing adjustments.

## What does this pull request do? (Please be technical)

[core] Speedup of LoadNPCList() and LoadMOBList() (Zach2good)

## Steps to test these changes

Load up your server, see that it loads faster and all mobs and NPCs spawn as expected

## Special Deployment Considerations
